woocommerce/woocommerce
# add tracks event for clicks on Manage suggestions link(s)

### All Submissions:

### Changes proposed in this Pull Request:
Adds usage tracking event for clicks on "Manage suggestions". For sites that opt-in, this will help us understand how site admins are interacting with Marketplace Suggestions.

### How to test the changes in this Pull Request:
1. Ensure `Usage tracking` is enabled – opt-in checkbox in [WooCommerce > Settings](http://localhost:6322/wp-admin/admin.php?page=wc-settings&tab=account)
1. Edit a product, scroll down to woo metabox. Click suggestions tab. 
1. Open dev tools network tab, click `Preserve log` (Chrome) or similar to preserve the network tab state across navigation. Optional: clear the network log, so the tracks request will be first.
3. Click `Manage suggestions` link in footer or in no-suggestions state. In network you should see a request for the tracks event. 

<img width="834" alt="Screen Shot 2019-04-09 at 1 59 02 PM" src="https://user-images.githubusercontent.com/4167300/55768148-c452c680-5acf-11e9-9550-66dc006efe3f.png">


### Other information:
